libs/wingsdk/src/target-tf-azure/app.ts

```diff
@@ -0,0 +1,55 @@
+import { AzurermProvider } from "@cdktf/provider-azurerm/lib/provider";
+import { IConstruct } from "constructs";
+// import { Polycons } from "polycons";
+import { IApp, CdktfApp, AppProps } from "../core";
+import { PolyconFactory } from "./factory";
+import { APP_AZURE_TF_SYMBOL } from "./internal";
+
+/**
+ * Azure app props
+ */
+export interface AzureAppProps extends AppProps {
+  /** Location for resources to be deployed to */
```

Comment:
Are there any docs listing the possible locations that we could link to in the docstring?